Apps/GatewayApi/src/Models/UserNote.cs

```diff
@@ -77,70 +77,5 @@ public class UserNote
         /// This is generally set by the baseDbContext.
         /// </summary>
         public string UpdatedBy { get; set; } = null!;
-
-        /// <summary>
-        /// Constructs a UserNote model from a Node database model.
-        /// </summary>
-        /// <param name="model">The note database model.</param>
-        /// <param name="cryptoDelegate">Crypto delegate to decrypt note.</param>
-        /// <param name="key">The security key.</param>
-        /// <returns>The user note model.</returns>
-        public static UserNote CreateFromDbModel(Database.Models.Note model, ICryptoDelegate cryptoDelegate, string key)
-        {
-            return new UserNote()
-            {
-                Id = model.Id,
-                HdId = model.HdId,
-                JournalDate = model.JournalDate,
-                Version = model.Version,
-                CreatedDateTime = model.CreatedDateTime,
-                CreatedBy = model.CreatedBy,
-                UpdatedDateTime = model.UpdatedDateTime,
-                UpdatedBy = model.UpdatedBy,
-                Title = model.Title != null ? cryptoDelegate.Decrypt(key, model.Title) : string.Empty,
-                Text = model.Text != null ? cryptoDelegate.Decrypt(key, model.Text) : string.Empty,
```

Comment:
It looks like NoteMapUtils only handles encrypting and decrypting the Text property and not the Title property.